pkg/project/project.go

```diff
@@ -1,7 +1,7 @@
 package project
 
 var (
-	bundleVersion = "0.23.1"
+	bundleVersion = "0.24.0-dev"
```

Comment:
Already know it will not be a patch, but minor?

Reply:
We can back into patch version whenever we need it for next time 😄

Reply:
The docs say it should be a patch version. We upgrade to minor version later if necessary.

https://github.com/giantswarm/giantswarm/blob/master/areas/empowerment/versioning-of-operators/how-to-release-an-operator.md#change-the-operator-version-in-the-code

@tomahawk28 Please do another PR so this is correct.